pkg/login/interactive_login.go

```diff
@@ -150,7 +150,7 @@ func protectTerminalState() (chan os.Signal, error) {
 		return nil, err
 	}
 
-	signalChan := make(chan os.Signal)
+	signalChan := make(chan os.Signal, 1)
```

Comment:
This was the linter complaining, I saw a fix here: https://youtrack.jetbrains.com/issue/GO-11092